src/Renderer/ThreeExtended/FeaturesUtils.js

```diff
@@ -0,0 +1,114 @@
+function overLine(point, vs, precision) {
```

Comment:
What is `vs`?
I'm all for short variable names in small scope (e.g: `for(let i ...)`) but function arguments should have explicit names.

Reply:
Try to stick to the convention and being method names with a verb please.